docarray/array/storage/weaviate/backend.py

```diff
@@ -65,6 +65,11 @@ def _init_storage(
         self._n_dim = config.n_dim
         self._serialize_config = config.serialize_config
 
+        if config.name and config.name != config.name.capitalize():
+            raise ValueError('weaviate class name has to be capitalized')
```

Comment:
Given this error report, the user would still not know how to correct the configurations. Could you give more clues in the output?

Reply:
Yep, addressed thank you.